### PR TITLE
Fix: DAO overview refactoring and multiple copy changes

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description

- Changed cards in DAO Overview page(Dao Creation) to carousel on mobile and changed header on the same page
- Changed copy on Create DAO pages
- Added link to "How To" on explore page
- IlluObject import refactored

Task: [APP-391](https://aragonassociation.atlassian.net/browse/APP-391), [APP-490](https://aragonassociation.atlassian.net/browse/APP-490), [APP-627](https://aragonassociation.atlassian.net/browse/APP-627)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/containers/carousel/index.tsx [hunk 1]
```diff
@@ -20,6 +20,11 @@ const Carousel: React.FC = () => {
   // when the rest of CTAs are enabled
   const handleCTAClick = useCallback(
     (path: string) => {
+      if (path.indexOf('://') !== -1) {
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/carousel/index.tsx]: Could you add a quick comment for what this doe?

File: packages/web-app/src/containers/daoOverview/index.tsx [hunk 2]
```diff
@@ -1,73 +1,134 @@
...
-    <div className="p-6 bg-ui-0 tablet:rounded-xl">
-      <h1 className="text-3xl font-bold text-ui-800">
-        {t('createDAO.overview.title')}
-      </h1>
-      <p className="mt-2 text-lg text-ui-600">
-        {t('createDAO.overview.description')}
-      </p>
+    <div className="tablet:flex items-end p-2 tablet:p-6 tablet:space-x-6 tablet:rounded-xl bg-ui-0">
+      <div>
+        <h1 className="font-bold ft-text-3xl text-ui-800">
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/daoOverview/index.tsx]: Nice, thanks for keeping in mind to adapt the fluid text.

**Review by WissenIstNacht — APPROVED** [review]
Hey Rakesh, thanks for the work. Just add the comment regarding the carousel into the source code.

File: packages/web-app/src/containers/carousel/index.tsx (hunk 1 above)
Comment (RakeshUP) [reply packages/web-app/src/containers/carousel/index.tsx]: This checks if the path is to a separate domain, and if so opens it in a new tab using `window.open()`, paths to the web-app pages are using `navigate`.
 
Changing this to `if(path.startsWith('http'))` for more brevity

**Review by sepehr2github — APPROVED** [review]
Looks good to me. Thanks, Rakash!

**Pull request merged**